docs/src/infra/protocols/telnet.md

```diff
@@ -1,6 +1,144 @@
 ---
-authors: ShutdownRepo
+authors: ShutdownRepo, Tednoob17
 ---
 
 # 🛠️ Telnet
 
+## Theory
+Telnet (teletype network) is a  network protocol used to gain access to virtual terminal in local or in remote systems . He provide  bidirectional text-based communication .
+
+----
+### Common Telnet Commands
+
+| Command   | Description                                                    |
+| --------- | -------------------------------------------------------------- |
+| `open`    | Connects to a specified local/remote host                      |
+| `close`   | Closes the current connection                                  |
+| `quit`    | Exits telnet                                                   |
+| `status`  | Shows the current status of the telnet client                  |
+| `z`       | Suspends telnet (on Unix/Linux systems)                        |
+| `set`     | Sets Telnet options (like terminal type)                       |
+| `unset`   | Unsets Telnet options                                          |
+| `display` | Displays current settings of Telnet options                    |
+| `send`    | Sends special characters or sequences (like break)             |
+| `mode`    | Sets the mode of operation (e.g., line by line or character)   |
+| `logout`  | Logs out from the remote system (not available on all systems) |
+
+
+## Practice
+### Enumeration
+#### Banner Grabbing
+To initiate a connection with telnet server and get any information about the target .
+
+
+> [!TIP]
+> The $TARGET_PORT is optional, but the default port is 23
+
+
+#### CLI tool
+::: tabs
+
+=== Unix-like
+
+```bash
+#Using Netcat
+nc -nv $TARGET_IP $TARGET_PORT
+#Using Telnet
+telnet $TARGET_IP $TARGET_PORT
+#Using Shodan-cli
+shodan stream --ports 23,1023,2323 --datadir telnet-data/ --limit 10000
+#Using nmap
+nmap -p  $TARGET_PORT -sVC  --script "*telnet* and safe" $TARGET_IP
+```
+
+
+=== Windows
+
```

Comment:
Can you add: "From Windows systems [...]" just like the comment on the unix part?